packages/opc-hello-world/README.md

```diff
@@ -0,0 +1,41 @@
+# This is opc-hello-world 👋
```

Comment:
Why do we have this?

Reply:
I don't think we should push this hello world component as an example. I suggest to push an actual exaxmple component. If you don't have anything ready u can push the generator code without the example component for now. As soon as ur generator PR is merged we can create the smallest component (Actual component) and push that

Reply:
Yes that sounds perfect. We can have the back-to.top componenet